integration/presto/src/main/java/org/apache/carbondata/presto/CarbondataPageSourceProvider.java

```diff
@@ -79,13 +80,31 @@
   @Override
   public ConnectorPageSource createPageSource(ConnectorTransactionHandle transactionHandle,
       ConnectorSession session, ConnectorSplit split, List<ColumnHandle> columns) {
-    this.queryId = ((CarbondataSplit)split).getQueryId();
+    CarbondataSplit carbondataSplit = (CarbondataSplit) split;
```

Comment:
Not expecting any changes in presto implementations, Changes should be only present in `PrestoCarbonVectorizedRecordReader` or `org.apache.carbondata.presto.CarbonVectorBatch`. Because these are reader classes why changes needed here.

Reply:
already reverted the modification. but it is difficult to reuse PrestoCarbonVectorizedRecordReader for streaming file.